gprofiler/main.py

```diff
@@ -941,9 +974,28 @@ def main() -> None:
             )
             sys.exit(1)
 
+        client_kwargs = {}
+        if args.upload_results:
+            if "server_upload_timeout" in args:
+                client_kwargs["timeout"] = args.server_upload_timeout
+            api_client = APIClient(
```

Comment:
`ProfilerAPIClient` accepts host, token, service and here it's mixed, keep same order pls